src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -225,14 +225,14 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   });
 
   return (
-    <div className="NavBarWrapper">
+    <div className="NavBarSchedule__wrapper">
```

Comment:
Tend to keep the original block class unaltered in case outside component/scss needs to make adjustments. This will relieve the need of guesswork since top component will always have class named the same as the component.

```suggestion
    <div className="NavBarSchedule NavBarSchedule__wrapper">
```

Reply:
fixed in 2c16847b0928ba957a737453f7b445db887dda23